package.json

```diff
@@ -43,5 +43,11 @@
     "yarn": "use pnpm",
     "npm": "use pnpm"
   },
+  "pnpm": {
+    "overrides": {
+      "@types/react": "npm:types-react@19.0.0-rc.1",
```

Comment:
I wonder if you should do this in staking/package.json or change this to do what you do there? Not sure 🤔

Reply:
overrides have to be in the root package.json as they apply to the resolution for all subpackages

Reply:
Oh really, good to know. Resolving